lib/src/tileset.dart

```diff
@@ -29,17 +36,22 @@ class TilesetPreprocessor {
   /// Pre-processes a tileset to eliminate some expensive processing from
   /// the rendering stage.
   ///
-  /// returns a pre-processed tileset
-  Tileset preprocess(Tileset tileset) {
-    theme.layers.whereType<DefaultLayer>().forEach((themeLayer) {
-      themeLayer.selector.select(tileset).forEach((layer) {
-        themeLayer.selector.layerSelector
-            .features(layer.features)
-            .forEach((feature) {
-          feature.decodeGeometry();
-        });
-      });
-    });
-    return Tileset._preprocessed(tileset);
+  /// The [themeLayerFeatures] options controls whether for each theme layer,
```

Comment:
Could we do without this? I'd prefer to avoid additional options if possible, since consumers will have to know what it's for and decide when to use it.

How about preprocessing always applying caching?

Reply:
Makes sense. The expectation is already that we trade memory for speed be preprocessing.